common/ui_common.c

```diff
@@ -879,3 +879,12 @@ void update_network_status(lv_obj_t *ui_staNetwork, struct theme_config *theme)
 
     if (file_exist(image_path)) lv_img_set_src(ui_staNetwork, image_embed);
 }
+
+void toast_message(const char * msg, uint32_t delay, uint32_t fade_duration) {
```

Comment:
the fade message that was added for adding and removing favourites was breaking all other messages in muxpore.  refactored to use this new toast_message method for displaying a message that fades out.  passing in a delay and fade_duration of 0 will just display the message without fading